src/applications/caregivers/components/PreSubmitInfo/SubmitLoadingIndicator.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { connect } from 'react-redux';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
+
+const SubmitLoadingIndicator = ({ submission }) => {
+  const [isLoading, setLoading] = useState(false);
+  const [timer, setTimer] = useState(0);
+  const [loadingMessage, setLoadingMessage] = useState(null);
+  const hasAttemptedSubmit = submission.hasAttemptedSubmit;
+  const isSubmitPending = submission.status === 'submitPending';
+
+  const getLoadingMessage = useCallback(
+    () => {
+      if (timer >= 0) {
+        setLoadingMessage('Please wait while we process your application.');
+      }
+
+      if (timer >= 15) {
+        setLoadingMessage(
+          'We’re processing your application. This may take up to 1 minute. Please don’t refresh your browser.',
+        );
+      }
```

Comment:
I haven't used `useCallback` that much, but it seems like `setLoadingMessage` would always get called twice... would it be better to add an `else` to the `timer >= 15` and set the loading message for `>= 0`?